File: feh.py
import os


def main():
    strn = 'feh --randomize --bg-fill ~/Imager/ms/output_{}.jpg'
    while 1:
        n = 1
        while (n <= 795):
            st = "{}".format(n)
            os.system(strn.format(st))
            n += 1


if __name__ == "__main__":
    main()

File: src/feh.cpp
#include<iostream>
#include<unistd.h>
#include<cstdlib>
#include<cstring>
int main(int argc,char *argv[])
{
	char *str = new(char[50]);
	while(1)
		for(int n = 0;n <= 795;n ++)
		{
			usleep(25000);
			sprintf(str,"feh --randomize --bg-fill ~/Imager/ms/output_%d.jpg",n);
			system(str);
		}
	delete [] str;
	return 	0;
}
